src/WultraMobileToken.ts

```diff
@@ -18,77 +18,63 @@ import { WMTOperations } from './operations/WMTOperations'
-     * @param pushBaseURL In case that the push server is on different URL.
-     *                    This is a rare scenario which doesn't happen in regular setup.
-     *                    When not set, baseURL is used.
-     * @param inboxBaseURL In case that the inbox server is on different URL.
-     *                     This is a rare scenario which doesn't happen in regular setup.
-     *                     When not set, baseURL is used.
+     * @param acceptLanguage Optionally sets the accept language for the outgoing requests headers for `operations`, `push` and `inbox` objects.
+     *                       The default value is "en".
+     *                       The value can be further modified in the each service object individualy.
+     *                       Standard RFC "Accept-Language" https://tools.ietf.org/html/rfc7231#section-5.3.5
+     *                       Response texts are based on this setting. For example when "de" is set, server
+     *                       will return operation texts in german (if available).
+     * @param userAgent Optionally sets the User agent that will be used in a HTTP hader. 
+     *                  Note that user-agent can be overriden by request processor in each API call.
      */
-    constructor(powerAuth: PowerAuth, baseURL?: string, pushBaseURL?: string, inboxBaseURL?: string) {
+    constructor(powerAuth: PowerAuth, acceptLanguage?: string, userAgent?: WMTUserAgent | string) {
 
-        let enrollmentURL = baseURL ?? powerAuth.configuration?.baseEndpointUrl ?? ""
-        let pushURL = pushBaseURL ?? enrollmentURL
-        let inboxURL = inboxBaseURL ?? enrollmentURL
+        // Retrieve the base URL and instantiate mtoken services.
+        let baseURL = powerAuth.configuration?.baseEndpointUrl ?? ""
 
-        this.operations = new WMTOperations(powerAuth, enrollmentURL)
-        this.push = new WMTPush(powerAuth, pushURL)
-        this.inbox = new WMTInbox(powerAuth, inboxURL)
-
-        WMTLogger.debug("Mobile Token object created with:")
-        WMTLogger.debug(" - baseURL: " + enrollmentURL)
-        WMTLogger.debug(" - pushURL: " + pushURL)
-        WMTLogger.debug(" - inboxURL: " + inboxURL)
-    }
+        this.operations = new WMTOperations(powerAuth, baseURL)
+        this.push = new WMTPush(powerAuth, baseURL)
+        this.inbox = new WMTInbox(powerAuth, baseURL)
 
-    /**
-     * Sets accept language for the outgoing requests headers for `operations`, `push` and `inbox` objects.
-     * 
-     * The value can be further modified in the each object individualy.
-     * 
-     * Default value is "en".
-     * 
-     * 
-     * Standard RFC "Accept-Language" https://tools.ietf.org/html/rfc7231#section-5.3.5
-     * Response texts are based on this setting. For example when "de" is set, server
-     * will return operation texts in german (if available).
-     */
-    setAcceptLanguage(lang: string) {
```

Comment:
`setAcceptLanguage` is present in the iOS and Android - why remove it here? The language can be changed during the runtime...

Reply:
Lol, thanks for this catch. I've overlooked it still exists in the natives. 😃 Reverted the change.

It can be, BTW, changed in runtime on the services separately, also to mimic the natives.